apps/minifront/src/components/dashboard/assets-table/index.tsx

```diff
@@ -43,7 +43,7 @@ export default function AssetsTable() {
           <TableHeader key={account.account} className='group'>
             <TableRow>
               <TableHead colSpan={2}>
-                <div className='flex max-w-full flex-col justify-center gap-2 md:flex-row pt-8 group-[:first-of-type]:pt-0'>
+                <div className='flex max-w-full flex-col justify-center gap-2 pt-8 group-[:first-of-type]:pt-0 md:flex-row'>
```

Comment:
This was just one of those tailwind class-order linting things. Figured I'd fix it while I was investigating this.